strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexerFactory.java

```diff
@@ -86,8 +87,8 @@ private IndexingContext createIndexingContext(String storageId,
         return indexerConfiguration.getIndexer()
                                    .createIndexingContext(storageId + ":" + repositoryId + ":" + indexType,
                                                           repositoryId,
-                                                          repositoryBasedir.toFile(),
-                                                          indexDir.toFile(),
+                                                          new File(repositoryBasedir.toUri().getPath()),
+                                                          new File(indexDir.toUri().getPath()),
```

Comment:
@sbespalov : Is this in line with the way things should be done with the file-system/path approach that we must follow?